force/api.go

```diff
@@ -178,6 +178,7 @@ type RestAPI interface {
 	UpdateSObject(id string, in SObject) (err error)
 	DeleteSObject(id string, in SObject) (err error)
 	GetSObjectByExternalId(id string, fields []string, out SObject) (err error)
+	GetSObjectBySpecificExternalId(id string, fields []string, specificExternalId string, out SObject) (err error)
```

Comment:
While you're here, can you rename `specificExternalId` -> `specificExternalType`